resource/ckb.toml

```diff
@@ -110,6 +110,62 @@ support_protocols = ["Ping", "Discovery", "Identify", "Feeler", "DisconnectMessa
 # [network.sync.header_map]
 # memory_limit = "256MB"
 
+# [network.proxy]
+### The proxy_url is user-specified SOCKS5 address, like: socks5://127.0.0.1:7891
+# proxy_url = "socks5://127.0.0.1:9050"
+
+### Toggle random authentication for the proxy server.
+### Default setting: true
+###
+### When enabled (true), CKB will generate a random username and password for each connection
+### to the proxy server.
+### For a Tor Server, this increases privacy by ensuring the proxy cannot correlate
+### multiple connections.
+### Refer to the 'IsolateSOCKSAuth' section in the following documentation:
+### https://manpages.debian.org/testing/tor/torrc.5.en.html
+###
+### Important Note:
+### If you are using a non-Tor SOCKS5 proxy that supports, but does not require authentication,
+### enabling this may lead to connection rejections. In such cases, set proxy_random_auth to false.
+# proxy_random_auth = true
+
+# [network.onion]
+### If CKB needs to run behind the Onion network,
+### the user should start a Tor server and set proxy_url to the Tor server's address.
+### then MUST set [network.proxy.proxy_url] to tor server's address.
+### The default Tor SOCKS5 proxy URL is socks5://127.0.0.1:9050.
+### For more information on configuring Tor, visit: https://linux.die.net/man/1/tor
+
```

Comment:
so the above description content is for `proxy_url`?
maybe it should moved to [networ.proxy] part?